module/dkg/broker.go

```diff
@@ -116,18 +120,22 @@ func (b *Broker) Broadcast(data []byte) {
 	}
 	maxedExpRetry := retry.WithMaxRetries(retryMax, expRetry)
 
-	err = retry.Do(context.Background(), maxedExpRetry, func(ctx context.Context) error {
-		err := b.dkgContractClient.Broadcast(bcastMsg)
+	f := func() {
+		err = retry.Do(context.Background(), maxedExpRetry, func(ctx context.Context) error {
+			err := b.dkgContractClient.Broadcast(bcastMsg)
+			if err != nil {
+				b.log.Error().Err(err).Msg("error broadcasting DKG result, retrying")
+			}
+			return retry.RetryableError(err)
+		})
+
 		if err != nil {
-			b.log.Error().Err(err).Msg("error broadcasting DKG result, retrying")
+			b.log.Error().Err(err).Msg("failed to broadcast message")
```

Comment:
same here: please at least include some goDoc why this is not a violation of our "do _not_ "just" log errors and continue" policy.